bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -247,7 +247,7 @@ spec:
           readinessProbe:
             exec:
               {{- if not .Values.service.tls.disable_base_client_port }}
-              command: ['/bin/bash', '-c', 'echo "ruok" | nc -w {{ .Values.readinessProbe.probeCommandTimeout }} -q {{ .Values.readinessProbe.probeCommandTimeout }} localhost {{ .Values.service.port }} | grep imok']
+              command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} nc -w {{ .Values.readinessProbe.probeCommandTimeout }} localhost {{ .Values.service.port }} | grep imok']
```

Comment:
apologies for the late comment but it seems this is using the wrong property, shouldn't it use `.Values.readinessProbe.probeCommandTimeout` instead of `.Values.livenessProbe.probeCommandTimeout` ?

Reply:
Hi @ddcprg your right, please check #3321

Reply:
hi @dani8art thanks for fixing this